images.go

```diff
@@ -29,7 +29,7 @@ const (
 type Image struct {
 	ID                string                 `json:"id"`
 	Filename          string                 `json:"filename"`
-	Metadata          map[string]interface{} `json:"metadata,omitempty"`
+	Metadata          map[string]interface{} `json:"meta,omitempty"`
```

Comment:
let's rename this field to `Meta` to match the API response

Reply:
Should I categorize this as a breaking change if this would break builds of applications referencing the old field?

Reply:
yeah, that's fine to update the changelog entry.